### PR TITLE
Add support for msgpack binary message format

### PR DESCRIPTION
**Pull request by kaashyapan**
Add few tests for protocol messages

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,90 +1,117 @@
+## [1.3.0]
+
```
Comment (sefidgaran) [inline CHANGELOG.md:2]: This branch has conflicts that must be resolved

**Review by sefidgaran — COMMENTED** [review]
This branch has conflicts that must be resolved

**Comment by sefidgaran** [comment]
Hi @kaashyapan ,

Thanks for the PR, I can see that after this PR the package in pub.dev is not showing WEB as supporte platform. Please check. thanks!
image

image

**Comment by kaashyapan** [comment]
Hmm. I was under the impression that the following commit made it possible for the msgpack library to be transpiled to js.
https://github.com/knopp/msgpack_dart/pull/9
I will need to dig deeper...

**Pull request merged**